core/drand_daemon.go

```diff
@@ -65,10 +67,11 @@ func NewDrandDaemon(c *Config) (*DrandDaemon, error) {
 		}
 
 		drandDaemon.state.Lock()
-		bp, isPresent := drandDaemon.beaconProcesses[beaconID]
+		bp, isPresent := drandDaemon.bpByID[beaconID]
 		drandDaemon.state.Unlock()
 
 		if isPresent {
```

Comment:
would it be worth adding a log line in cases where the dkgCallback is fired and we don't find a beacon for the requested group?